### PR TITLE
Mayhem integration

### PR DESCRIPTION
**Pull request by Verylostpenguin**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+project: Verylostpenguin/smoltcp
+target: smoltcp
+duration: 90
+
+cmds:
+  - cmd: /smoltcp/fuzz/target/x86_64-unknown-linux-gnu/release/dhcp_header
```
Comment (nathanjackson) [inline Mayhemfile:6]: Similar to your other PR. We need changes to your Mayhemfile.

- Remove your username from the project.
- No need to include duration.
- Please remove all but one cmd and rename the target field to reflect it. Mayhemfiles should be written for one target, but the multiple commands listed here would actually be for multiple targets.

**Review by nathanjackson — CHANGES_REQUESTED** [review]
Thanks for your PR.

See my comments, there are a few things we need to have fixed before we can merge this.

File: Mayhemfile (hunk 1 above)
Comment (Verylostpenguin) [reply Mayhemfile]: I've deleted the username and duration from the mayhemfile. I also created multiple mayhem files which I run in the yml file sequentially. I hope this addresses all the issues listed above.

**Review by nathanjackson — APPROVED** [review]
Looks good to me.

**Comment by nathanjackson** [comment]
@Verylostpenguin Thanks for your integration, someone should be reaching out to you about the $750 payment.

I am going to turn on automatic updates to this repo. Please keep an eye on this repo to ensure the Mayhem action continues to work. If something breaks within the next 30 days, we'll look to you to fix it for the last $250 on this submission.

**Pull request merged**